### ISSUE TITLE
Research Permit Resource Model edits

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#898]
- [x] ~Under Research Location, at bottom of page, add a multi-pick option "within Provincial Park" and "within other parks"~ Replace "in a park" research assessment criteria with "within provincial park" and "within other park"
- [x] Under Research Location, at bottom of page, add a large freeform textbox to accommodate a list of up to 12 names, called "First Nations Consultation Areas"
- [x] Under Research Permit Assessment Criteria, change "on park" to "in a park"
- [x] On Final Research Report FMO Assessment page – final header should say “FMO Report Response”

Questions:
- [x] Can we create multiple Communication Logs – one for each First Nation? 
- [x] Will First Nations Contact be associated with all information as entered as Contributor? - Yes

**Comment by bferguso** [comment]
@LeannePyle  - Just wondering to simplify things - can we just change the Research Assessment criteria from "in a park" to within Provincial Park" and "within other parks" so we don't have to add another pick list?

**Comment by LeannePyle** [comment]
Yes, I agree that is a better spot for this information -adding these options under RP Assessment Criteria

**Comment by bferguso** [comment]
Automated cleanup
